app/views/unattended/provisioning_templates/PXEGrub/pxegrub_default_local_boot.erb

```diff
@@ -4,8 +4,18 @@ name: PXEGrub default local boot
 model: ProvisioningTemplate
 %>
 <%# Used to boot provisioned hosts, do not associate or change the name. %>
-
-default=0
+<%
+  # Grub1 only supports numeric default statement, this allows conversion to number. First define
+  # array of directories we will search for EFI booloaders:
+  paths = ["fedora", "redhat", "centos", "debian", "ubuntu", "sles", "opensuse", "Microsoft", "EFI"]
+  # Add remaining entries to it and use this to convert to number:
+  items = paths.push("local_chain_hd0")
+  # Read default setting but since "local" is missing, use the first path available.
+  default_setting = global_setting("default_pxe_item_global", paths.first)
```

Comment:
This has been already fixed in previous commits, to use `default_pxe_item_local` instead of `default_pxe_item_global` (as I was commenting on it in https://github.com/theforeman/foreman/pull/6052#discussion_r219328178). Was there a reason to turn it back to global, or just a mistake?